### PR TITLE
Increase title margin on small screens

### LINKED ISSUE #3123: The title on matrix are to close on small screens
## Pattern to amend
`p-matrix`

## Visual
vanilla-matrix-tight-header

## Context
The title is given `margin-bottom: 0.2rem !important;` on small screens which seems to tight.

### PR DESCRIPTION
**Pull request by lyubomir-popov**
## Done

Increase title margin on small screens
Fixes #3123

## QA

- Pull code
- Run `./run`
- Open /docs/examples/patterns/matrix
- Verify margin is increased on small screens

## Screenshots

image

image

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
@spencerbygraves can review percy please?

**Comment by lyubomir-popov** [comment]
@spencerbygraves the misaligned icon is not a regression of this pr, if you open this page in firefox ansd resize to a mobile width it happens with the currently released version of vanilla too:

image


Can we merge this and resolve the misalignment in [this separate issue](https://github.com/canonical-web-and-design/vanilla-framework/issues/3191)?

**Comment by lyubomir-popov** [comment]
@spencerbygraves I've fixed the icon on this, can you review percy again please:

image

**Comment by spencerbygraves** [comment]
@lyubomir-popov done 👍

File: scss/_patterns_matrix.scss [hunk 1]
```diff
@@ -121,7 +122,16 @@
 
   .p-matrix__title {
     @extend %vf-heading-4;
-    @extend %u-no-margin--bottom--h4;
+
+    @media (min-width: $breakpoint-heading-threshold) {
+      margin-bottom: -#{map-get($nudges, nudge--h4)} !important;
```
Comment (bartaz) [inline scss/_patterns_matrix.scss:127]: Why is `!important` needed here? Class name styling should give enough specificity to override h* styling.

This applies to line 132 as well.

**Comment by lyubomir-popov** [comment]
@bartaz doesn't seem to be needed, removed.

**Comment by bartaz** [comment]
@lyubomir-popov I think you didn't push the commit with removed `!important`

**Comment by lyubomir-popov** [comment]
sorry, pushed again.

**Pull request merged**